testing/py-protobuf/APKBUILD

```diff
@@ -0,0 +1,28 @@
+# Maintainer: Corentin Henry <corentinhenry@gmail.com>
+pkgname=py-protobuf
+_pkgname=protobuf
+pkgver=2.6.1
+pkgrel=0
+pkgdesc="Protocol Buffers are Google’s data interchange format."
+url="https://github.com/google/protobuf"
+arch="noarch"
+license="Copyright 2014, Google Inc. All rights reserved."
```

Comment:
@jirutka I don't see mention of the BSD license. I copy/pasted the first line of the license file : `Copyright 2014, Google Inc. All rights reserved.`.

I don't have much licensing knowledge, how do you know this is a BSD license?